bookkeeper-server/src/main/java/org/apache/bookkeeper/http/JsonUtil.java

```diff
@@ -0,0 +1,53 @@
+/**
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+package org.apache.bookkeeper.http;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+/**
+ * Util class which helps to parse Json to and from Object.
+ */
+public class JsonUtil {
+
+    public static String toJson(Object object) throws ParseJsonException {
+        ObjectMapper mapper = new ObjectMapper();
```

Comment:
Object Mapper is threadsafe and usually it can be kept as a static field. It cachea the structure of classes and so it is more performant to keep only one instance please use only a static field